conda-store-server/conda_store_server/server/app.py

```diff
@@ -119,6 +122,9 @@ def initialize(self, *args, **kwargs):
         super().initialize(*args, **kwargs)
         self.load_config_file(self.config_file)
 
+        if self.config.CondaStore.upgrade_db:
+            dbutil.upgrade(self.config.CondaStore.database_url)
+
```

Comment:
Move to below the `self.conda_store = ...` bits and then use `if self.conda_store.upgrade_db: ....` instead. This ensures that CondaStore Configuration object is properly initialized.

Reply:
Done.